submissions/amooChristianah/challenge2/solution2.js

```diff
@@ -0,0 +1,12 @@
+function solution3(nums) {
```

Comment:
Nice approach.
However, this is the solution for the third question and the file is named solution2. Please rectify that.

This is an O(n^2) solution. You could run into Time Limit issues for specific test cases. You could try to solve it in a better time complexity. 
Sorting and [dictionaries](https://stackoverflow.com/a/11789703) are approaches that may be of help.